tmux/2022-11-PI_Eval1-coffee-from_bag_and_truth.yml

```diff
@@ -109,4 +109,5 @@ windows:
       -p skip_score_threshold:=300.0
 
   # Currently hard-coded to use the /debug ROS namespace.
-  - engineering_ui: ros2 launch src/angel_utils/demo_ui/launch/websocket.launch
+  - engineering_ui_websocket: ros2 launch rosbridge_server rosbridge_websocket_launch.xml port:=9090
```

Comment:
Should `angel_utils/demo_ui` also be renamed?

Reply:
It could be, but I'm happy to bikeshed that tomorrow at stand-up vs. now.
I was considering that it also maybe doesn't belong in angel_utils, but I also see why it's there, so eh.